README.adoc

```diff
@@ -4,6 +4,34 @@ Configure Amazon Web Services (AWS) Identity and Access Management (IAM) credent
 This action implements the AWS SDK credential resolution chain and sets configuration and credential files for other CloudBees actions to use.
 Configuration and credential files are detected by both the AWS SDKs and the AWS CLI for AWS API calls.
 
+== Additional information about `aws-session-token`
+
+The `aws-session-token` is used in cases where temporary session credentials are required. AWS recommends using temporary credentials as a best practice to enhance security. Most users will need to provide a session token, as it is typically required for credentials issued through role assumption or federated authentication mechanisms.
```

Comment:
I would advise recommending users to use OIDC authentication rather than supply the api key, secret key and session token.

The session token is typically unworkable as you cannot practicably store it in a secret as you will need to continually update the secret by hand. The use case for this session token is if you are doing complex workflows that are typically better handled by leveraging the OIDC token issued by CloudBees

Reply:
Changed it: 
The `aws-session-token` is used when temporary session credentials are needed. However, users are encouraged to use OIDC authentication instead, as it simplifies credential management and avoids the need to update session tokens manually. The `aws-session-token` is best suited for complex workflows where OIDC authentication may not be an option, though OIDC is typically the preferred approach.